bears/js/PrettierLintBear.py

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
Why leave this line?

Reply:
Must be an accidental press. I will remove it.